Sources/UITouch.swift

```diff
@@ -53,8 +53,6 @@ public class UITouch {
             action(recognizer)
         }
     }
-
-    internal var hasBeenCancelledByAGestureRecognizer = false
```

Comment:
I don't know why we decided to change the computed to a stored property, maybe the stored prop has advantages I'm currently missing. I started my work by reverting https://github.com/flowkey/UIKit-cross-platform/commit/9a698b9914e62e4623704ee0fcefb14602b88e9c which introduced the buggy behaviour. 

It was already kinda buggy even before this commit, but it was usable

Reply:
@michaelknoch can you confirm that the issue #305 was trying to solve is still fixed following the change here? I can see that @janek added tests for that at the time but I don't have a clear picture whether they would have failed before that change. You could find out by reverting that PR except for the tests, and then running the tests (they should fail in that state, and then pass with this PR's changes)

Reply:
the tests still pass, and they didn't fail because touchesEnded is still working correctly (not being called when there is a recognizer) - but in our case touchesBegan is the problematic one because it still does get called even when there are recognizers (which hides the draghandles), but since touchesEnded was fixed the dragHandles did not appear again on touch up

not sure if this made any sense. in short: improving touchesEnded behaviour without improving touchesBegan behaviour introduced the bug in our player. this pr brings these behaviours together again. To answer your question: I think it's fine, I also enhanced the existing tests